### DISCUSSION TITLE
Validity of geographic associations in the Synthea data

### DISCUSSION THREAD

**Discussion by PhilipG7** [synthetichealth/synthea#1506] [General]
Hello,

We would like to use the Synthea data for analyses that explore geographic associations between demographics and health outcomes. We noticed that the addresses that are generated for the patient file are not valid addresses. 

To what extent do associations between the patients' zip code, age, race, sex, and income mirror the real-world geographic distribution of those variables, and to what extent do associations between these demographic characteristics and health outcomes hold up? What input files are used to establish these associations if they exist?

Thank you.

**Comment by jawalonoski** [comment]
The addresses are fake, so that you can't find a synthetic patient with HIV and say "_hey someone with HIV lives at my neighbor's house!_" I hope you can see why something similar to that might be problematic.

The SDoH and other demographic distributions match the "named locations" (e.g., city, town, etc) present in the various demographic and associated files. See https://github.com/synthetichealth/synthea/wiki/Default-Demographic-Data and https://github.com/synthetichealth/synthea/wiki/Demographics-for-Other-Areas#demographics-file-format

The extent that these variables impact each patient's life history depends on how each module takes these attributes into account, which varies module by module. We don't have a trace available to illustrate how these attributes impact various diseases. So, the best you can do right now is view the modules you're interested in with the Module Builder (https://synthetichealth.github.io/module-builder/) or run `./gradlew attributes` and view the `./output/attributes.json` to see what modules read/write which attributes.

**Reply by PhilipG7** [reply]
Thank you for the response.

I see that the demographic files in the default demographic data folder include education and income distributions from the ACS. However, the links to these files appear to be dead, and I wasn't able to locate them anywhere on the GitHub page. 

Are these files at the zip-code level, such that the Synthea data would include valid zip-code level income and education level distributions? Is there a way to locate these input files on the GitHub page so I could examine their composition?

**Reply by jawalonoski** [reply]
See https://github.com/synthetichealth/synthea/tree/master/src/main/resources/geography

**Reply by jawalonoski** [reply]
The original input files were transformed into those files. The original source files from the US Census Bureau and other sources are not included in the repository.
